Cannot run tests on Windows

```
z:\Documents\GitHub\tcomb-react>npm test

> tcomb-react@0.8.9 test z:\Documents\GitHub\tcomb-react
> npm run lint && node_modules/.bin/mocha


> tcomb-react@0.8.9 lint z:\Documents\GitHub\tcomb-react
> eslint index.js

'node_modules' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! Test failed.  See above for more details.
```
I have a fix for which I'll submit a PR soon.

--- a/package.json
+++ b/package.json
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint index.js",
-    "test": "npm run lint && node_modules/.bin/mocha"
+    "test": "npm run lint && npm run mocha",
+    "mocha": "mocha"
   },
   "repository": {
     "type": "git",

